src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js

```diff
@@ -146,25 +158,23 @@
             var turnIcon = Directions.getTurnIconName(position.properties.directions.turn);
             var speedMs = (ProfileService.getCurrentUser().preferences.speed || 1) * mphToMs;
             var timeMins = (distanceToTurn / speedMs / 60).toFixed(0);
-            var distanceText = 'In approx. ' +  timeMins +
-                ' min (' + $filter('distance')(distanceToTurn) + ')';
             if (position.properties.turnamenity) {
-                distanceText += ' at the ' + position.properties.turnamenity.name;
+                text += ' at the ' + position.properties.turnamenity.name;
             }
-            var subtitleText = distanceText;
-            var rightImages = [];
+            var images = [];
             if (position.properties.directions.warnings && position.properties.directions.warnings.length > 0) {
-                rightImages.push(position.properties.directions.warnings[0].img);
+                images.push(position.properties.directions.warnings[0].img);
```

Comment:
If there's no use for `rightImages` any more, might as well remove it from the Navbar properties, too.